torch/utils/data/datapipes/iter/combining.py

```diff
@@ -73,11 +72,10 @@ def __new__(cls, datapipe: IterDataPipe, num_instances: int, buffer_size: int =
 
 
 class _ForkerIterDataPipe(IterDataPipe):
```

Comment:
Just want to confirm this is a private class and not shown in our doc even though we have doc strings.